tahrir/views.py

```diff
@@ -227,7 +227,57 @@ def admin(request):
         if token != request.POST['csrf_token']:
             raise HTTPForbidden('CSRF token did not match')
 
-        if request.POST.get('add-person'):
+        if request.POST.get('add-team'):
+            team_name = request.POST.get('team-name')
+            team_id = convert_name_to_id(team_name)
+            if not request.db.team_exists(team_id=team_id):
+                request.db.create_team(name=team_name)
+                request.session.flash(
+                    "You created a team with name {0}".format(team_name))
+            else:
+                request.session.flash(
+                    "Team with name {0} already exists.".format(team_name))
+
+        elif request.POST.get('add-series'):
+            data = request.POST
+            series_name = data.get('series-name')
+            series_id = convert_name_to_id(series_name)
+            if not request.db.series_exists(series_id):
+                description = data.get('series-description')
+                tags = data.get('series-tags')
+                team_id = data.get('series-team-id')
+
+                request.db.create_series(
+                        series_id=series_id,
+                        name=series_name,
+                        desc=description,
+                        tags=tags,
+                        team_id=team_id)
+                request.session.flash(
+                    "You created a series with name {0}".format(series_name))
+            else:
+                request.session.flash(
+                    "Series with name {0} already exists.".format(series_name))
+
+        elif request.POST.get('add-milestone'):
+            data = request.POST
+            series_id = data.get('milestone-series-id')
+            badge_id = data.get('milestone-badge-id')
+            if not request.db.milestone_exists_for_badge_series(badge_id,
+                                                           series_id):
```

Comment:
You may want to split after the ``(``, with 1 indentation more than the logic bloc it should be valid pep8